packages/fxa-content-server/app/scripts/models/account.js

```diff
@@ -367,6 +367,34 @@ const Account = Backbone.Model.extend(
       });
     },
 
+    /**
+     * Fetches account details from GET /account, for use by the settings views.
+     * Caches its own result, because it's not intended for that endpoint to be
+     * called repeatedly like some of the polling methods are. If you're really,
+     * really sure you need to, pass the `force` option to force a clean request.
+     *
+     * @param {Object} [options]
+     *   @param {Boolean} [options.force] - Ignore any cached results
+     *
+     * @returns {Promise}
+     */
+    settingsData(options = {}) {
+      return Promise.resolve().then(() => {
+        if (this._settingsData && !options.force) {
```

Comment:
It seems like each of the fields in `this._settingsData` could be stored in the [account's attributes](https://github.com/mozilla/fxa/blob/c2e696500db9715eca8a42660c60c5a016335050/packages/fxa-content-server/app/scripts/models/account.js#L58) rather than stored as an instance variable? That would allow you to do things like `account.get('subscriptions')`. I can make a case either way, though storing model data in the model attributes is closer to how the models are meant to be used.

Reply:
Just making sure I understand the suggestion. Are you saying do that and still have this call return the aggregated result, or are you saying do that and then have callers call `account.get` for whichever properties it is they're interested in?

Reply:
I was saying the 2nd, though on reflection that doesn't seem like a good idea. Keep it like this.